imports/test-utils/__mocks__/meteor/alanning:roles.js

```diff
@@ -0,0 +1,13 @@
+export const Roles = {
```

Comment:
created the following files to get jest tests passing.
we tell jest to replace all `meteor/` modules with local mocks, and some were missing.